projectName-selenium/src/test/resources/META-INF/spring/selenium.properties

```diff
@@ -1,3 +1,9 @@
 selenium.serverUrl=http://localhost:8080
 selenium.contextName=projectName-web
 selenium.applicationContextUrl=${selenium.serverUrl}/${selenium.contextName}
+
+selenium.proxyUserName=
+selenium.proxyUserPassword=
+selenium.proxyHttpServer= 
+
+selenium.geckodriverVersion=0.14.0
```

Comment:
- [ ] Please write a comment "Please set the appropriate version according to the web browser."

Reply:
add comment.
b3667ca